spec/controllers/verify/address_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe Verify::AddressController do
+  def stub_subject(user)
+    user_session = {}
+    stub_sign_in(user)
+    idv_session = Idv::Session.new(user_session, user)
+    idv_session.resolution = Proofer::Resolution.new success: true, session_id: 'some-id'
+    idv_session.applicant = Proofer::Applicant.new first_name: 'Some', last_name: 'One'
+    idv_session.vendor = subject.idv_vendor.pick
+    allow(subject).to receive(:confirm_idv_session_started).and_return(true)
+    allow(subject).to receive(:confirm_idv_attempts_allowed).and_return(true)
+    allow(subject).to receive(:idv_session).and_return(idv_session)
+    allow(subject).to receive(:user_session).and_return(user_session)
```

Comment:
Would this test be better performed as a feature spec instead since all of this stuff needs to exist beforehand, and we might inadvertently set the wrong prerequisite values?